Smarty/templates/ComposeEmail.tpl

```diff
@@ -56,7 +56,7 @@
 	</td>
 </tr>
 	<tr>
-	<td class="mailSubHeader" align="right"><font color="red">*</font><b>{$MOD.LBL_FROM}</b></td>
+	<td class="mailSubHeader" align="right"><div style="color: red;">*</div><b>{$MOD.LBL_FROM}</b></td>
```

Comment:
can we follow LDS recommendations for this. Instead of just changing it, why don't we apply LDS now and avoid having to change it all again later?